documentation/images/plantuml/agent/flows/ap_metrics_cert_4.7.6.puml

```diff
@@ -0,0 +1,21 @@
+@startuml
+participant controller
```

Comment:
I didn't know about these declarations! #til

Reply:
I fell in love with PlantUML the first time I knew about. However, I decided not to memorize its syntax at all. When I have to create a diagram, I always open https://plantuml.com/ side by side and use the examples as a guide.